makiflow/layers/trainable_layers.py

```diff
@@ -851,8 +851,8 @@ def to_dict(self):
 class BatchNormLayer(BatchNormBaseLayer):
     TYPE = 'BatchNormLayer'
 
-    def __init__(self, D, name, decay=0.9, eps=1e-4, use_gamma=True,
-                 use_beta=True, mean=None, var=None, gamma=None, beta=None, track_running_stats=True):
+    def __init__(self, D, name, decay=0.9, eps=1e-4, use_gamma=True, use_beta=True, regularize_gamma=True,
+                 regularize_beta=True, mean=None, var=None, gamma=None, beta=None, track_running_stats=True):
```

Comment:
Set regularization for this parameter turned off by default. It is actually doesn't make sense to regularize these in case of normalization.